deploy/helm/sumologic/values.yaml

```diff
@@ -212,6 +212,7 @@ sumologic:
     ## ref: https://github.com/fabric8io/fluent-plugin-kubernetes_metadata_filter#configuration
     cacheTtl: "3600"
 
+    cacheRefresh: "600"
```

Comment:
can we add a comment explaining this and include `(in seconds)` like we did for TTL?